src/components/Common/SnapshotRedirectButton.css

```diff
@@ -0,0 +1,13 @@
+.SnapshotRedirectButton {
+  display: flex !important;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+  border: 1px solid rgba(115, 110, 125, 0.32) !important;
```

Comment:
Can we use --black-100 here?

Reply:
The thing is that it needs to be the same as the choice buttons border, which is slightly lighter. I'll ask Juani if we can add it to the theme or change both.